lib/rubocop/cops_documentation_generator.rb

```diff
@@ -33,14 +33,15 @@ def cops_of_department(department)
     cops.with_department(department).sort!
   end
 
-  def cops_body(cop, description, examples_objects, safety_objects, pars) # rubocop:disable Metrics/AbcSize
+  def cops_body(cop, description, examples_objects, safety_objects, see_objects, pars) # rubocop:disable Metrics/AbcSize, Metrics/ParameterLists
     check_examples_to_have_the_default_enforced_style!(examples_objects, cop)
 
     content = h2(cop.cop_name)
     content << required_ruby_version(cop)
     content << properties(cop)
     content << "#{description}\n"
     content << safety_object(safety_objects) if safety_objects.any? { |s| !s.text.blank? }
+    content << see_object(see_objects) if see_objects.any?
```

Comment:
[memo] I'm putting `@see` here because it often comes before `@example`[^1], but open to suggestions for other locations.

[^1]: Currently, RuboCop RSpec does so.